exporter/sentinels.go

```diff
@@ -66,6 +66,16 @@ func (e *Exporter) extractSentinelMetrics(ch chan<- prometheus.Metric, c redis.C
 		}
 		masterAddr := masterIp + ":" + masterPort
 
+		masterCkquorum, err := redis.String(doRedisCmd(c, "SENTINEL", "CKQUORUM", masterName))
```

Comment:
Make this `masterCkquorumMsg, err := ` and remove line 72? (keep 75)

Reply:
Make sense. On it

Reply:
🎉